lang/src/loader_account.rs

```diff
@@ -156,6 +156,17 @@ impl<'info, T: ZeroCopy + Owner> Key for AccountLoader<'info, T> {
     }
 }
 
+
+///
+/// # Safety
+/// 
+/// Dereferencing the self.data raw pointer is safe because it will always point to valid data.
+/// This is because after the instantiation of an AccountLoader
+/// the location of the data the pointer is pointing to (the memory location of the &[u8] inside the inner AccountInfo)
```

Comment:
Please make all comments within 80 chars. Not to be a stickler here, but this has a real impact on one's ability to read code when using an editor with split screens. Over 80 chars and the comment become unreadable without scrolling.

Reply:
great point, I will add that to my list of things to add to Contributing.md